packages/astro/src/vite-plugin-dev-toolbar/vite-plugin-dev-toolbar.ts

```diff
@@ -65,13 +65,16 @@ export default function astroDevToolbar({ settings, logger }: AstroPluginOptions
 				return `
 					export const loadDevToolbarApps = async () => {
 						return (await Promise.all([${settings.devToolbarApps
-							.map((plugin) => `safeLoadPlugin(${JSON.stringify(plugin)})`)
+							.map(
+								(plugin) =>
+									`safeLoadPlugin(async () => (await import(${JSON.stringify(plugin)})).default)`
```

Comment:
I think we also need to pass second parameter of `JSON.stringify(plugin)`, since on line 85 the `safeLoadPlugin` function references the `entrypoint` variable string

Reply:
This is why programming in strings was a mistake

Reply:
We can also do this:

```js
async function something() {
  // ...
}

// ...
return `
  ${something.toString()}
`
```

But either way works for me 😄